app/src/main/java/com/example/bigowlapp/utils/AlarmBroadcastReceiverManager.java

```diff
@@ -0,0 +1,120 @@
+package com.example.bigowlapp.utils;
+
+import android.app.AlarmManager;
+import android.app.PendingIntent;
+import android.content.Context;
+import android.content.Intent;
+import android.widget.Toast;
+
+import com.example.bigowlapp.model.Response;
+import com.example.bigowlapp.model.Schedule;
+import com.example.bigowlapp.repository.ScheduleRepository;
+import com.example.bigowlapp.service.AlarmBroadcastReceiver;
+import com.google.android.gms.tasks.Task;
+import com.google.android.gms.tasks.Tasks;
+import com.google.firebase.Timestamp;
+import com.google.firebase.firestore.GeoPoint;
+import com.google.firebase.firestore.QuerySnapshot;
+
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.List;
+
+import static com.example.bigowlapp.utils.IntentConstants.EXTRA_LATITUDE;
+import static com.example.bigowlapp.utils.IntentConstants.EXTRA_LONGITUDE;
+import static com.example.bigowlapp.utils.IntentConstants.EXTRA_UID;
+
+/**
+ * The purpose of this class is to set/define alarms that the app will set.
+ * After the time activation of an alarm, code will be executed from
+ * {@link com.example.bigowlapp.service.AlarmBroadcastReceiver}
+ */
+public class AlarmBroadcastReceiverManager {
```

Comment:
Might want to consider a name change to reflect the specific puprose of this alarm manager, especially in the case we need alarm for other things. Name it based on the purpose of this specific Alarm manager.